### ISSUE TITLE
the gradient function returned by tfe.implicit_value_and_gradients() doesn't support keyword argument

### ISSUE THREAD

**Issue by DHZS** [tensorflow/tensorflow#19718]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Windows 10
- **TensorFlow installed from (source or binary)**:
Binary
- **TensorFlow version (use command below)**:
v1.8.0
- **Python version**: 
Python 3.6.4
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
No
- **GPU model and memory**:
No
- **Exact command to reproduce**:

### Describe the problem
When using eager mode, I found that the gradient function returned by `tfe.implicit_value_and_gradients()` doesn't support keyword argument.

### Source code / logs
My code like this
```python
grad_fn = tfe.implicit_gradients(MLP)
grads_and_vars = grad_fn(data, label, is_training=True)
```

And I got the error
>Traceback (most recent call last):
  File "xxx.py", line 91, in <module>
    grads_and_vars = grad_fn(data, label, is_training=True)
  File "xxx\anaconda3\lib\site-packages\tensorflow\python\eager\backprop.py", line 289, in grad_fn
    return implicit_val_and_grad(f)(*args, **kwds)[1]
TypeError: grad_fn() got an unexpected keyword argument 'is_training'

When I remove the keyword "is_training", like this
```
grads_and_vars = grad_fn(data, label, True)
```
Then the code has no problem. 

The bug can be fixed by modifying ’xxx\Lib\site-packages\tensorflow\python\eager\backprop.py‘ and adding `**kwds` in function `implicit_val_and_grad(f)`
the modified code like this
```python
  def grad_fn(*args, **kwds):
    """Computes the gradient of the wrapped function."""
    this_tape = tape.push_new_tape()
    try:
      end_node = f(*args, **kwds)
  ...
```

**Comment by alextp** [comment]
I agree with your suggested fix. Do you want to send a pull request? (I could do it, but I think it's nicer if you get the credit for your contribution!)

**Comment by DHZS** [comment]
OK~ You can check it at here: https://github.com/tensorflow/tensorflow/pull/19804

**Comment by tensorflowbutler** [comment]
Nagging Assignee @alextp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by reedwm** [comment]
Looks like this was fixed by #19804.
